src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case ('addProperties'):
```

Comment:
you don't need brackets for cases
```suggestion
      case 'addProperties':
```

Reply:
> you don't need brackets for cases

Thanks, removed